PrivateCollections/Germany/Neukirchen006.xml

```diff
@@ -0,0 +1,502 @@
+                              <item xml:id="q8" n="7">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="53r" to="60v"/>
+                              </item>
+                              <item xml:id="q9" n="8">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="61r" to="68v"/>
+                              </item>
+                              <item xml:id="q10" n="9">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="69r" to="76v"/>
+                              </item>
+                              <item xml:id="q11" n="10">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="77r" to="84v"/>
+                              </item>
+                              <item xml:id="q12" n="11">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="85r" to="92v"/>
+                              </item>
+                              <item xml:id="q13" n="12">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="93r" to="100v"/>
+                              </item>
+                              <item xml:id="q14" n="13">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="101r" to="108v"/>
+                                 s.l. 3, stub after 5
+                                 s.l. 6, stub after 2
+                              </item>
+                              <item xml:id="q15" n="14">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="109r" to="116v"/>
+                              </item>
+
+                              <item xml:id="q16" n="15">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="117r" to="124v"/>
+                              </item>
+
+                              <item xml:id="q17" n="16">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="125r" to="140v"/>
+                                </item>
+
+                              <item xml:id="q18" n="17">
+                                 <dim unit="leaf">4</dim>
+                                 <locus from="141r" to="144v"/>
+                              </item>
+                           </list>
+                        </collation>
+
+                        <condition key="good">
+                          The first (endleaf) quire, <locus from="1r" to="4v"/> is added later,
+                           its two leaves are made of pieces of parchment
+                          carefully sewn in the middle (along the lines).
+                          Fols. <locus from="143" to="144"/> are damaged by insects.
```

Comment:
It seems to me that you are using fols./ff. interchangeably? When they are marked up with locus, it is not necessary to write this, because the app will provide it

Reply:
Ok